.golangci.yml

```diff
@@ -20,6 +20,12 @@ linters-settings:
     local-prefixes: github.com/cmsgov/easi-app
   revive:
     confidence: 0.1 # Default is 0.8. We want to ignore things that have an extremely low confidence level
+  exhaustive:
+    # Presence of "default" case in switch statements satisfies exhaustiveness,
+    # even if all enum members are not listed.
+    # Default: false
+    default-signifies-exhaustive: true
```

Comment:
this allows `default` in a switch case to satisfy the `exhaustive` linter.

the `exhaustive` linter will check if a switch case hits all possible enum values of the type being switched on. the `default` case normally does not satisfy that linter, so this allows for that (@ClayBenson94 @mynar7 - if this is not acceptable, I will remove. Please let me know!)

Reply:
I think Dylan flipped this to not allow default previously but I never paid that much attention to it to be honest.